app/services.js

```diff
@@ -19,29 +19,7 @@ airTrafficControlApp.service('systemBootService', ['$location', '$q', '$timeout'
     
     // Function is called when the system is booted
     this.bootSystem = function(){
-        // Async call
-        var deferred = $q.defer();
-        
-        // Resolve the deferred if the system
-        // is already booted
-        if (true === self.systemIsBooted()) {
-            deferred.resolve();
-        }
-        else {
-            // emulate async call
-            $timeout(function(){
-                // Set the system to booted status
-                self.systemBootStatus = 'Booted';
-                
-                // Resolve the deferred to indicate
-                // success
-                deferred.resolve(self.systemBootStatus);
-                
-            }, 50);
-        }
-        
-        // Make sure to return a promise from the async call
-        return deferred.promise;
+        self.systemBootStatus = 'Booted';

```

Comment:
what is bootSystem doing? I would have thought it clears the queue, but here it simply just sets a string?